client/src/localization/languages/Ar.tsx

```diff
@@ -0,0 +1,276 @@
+  com_endpoint_config_key_edge_full_key_string: ' لتقديم سلاسل كاملة للكوكيز.',
+  com_endpoint_config_key_chatgpt: 'للحصول على رمز الوصول الخاص بـ ChatGPT إصدار مجاني، قم بتسجيل الدخول إلى',
+  com_endpoint_config_key_chatgpt_then_visit: 'ثم قم بزيارة',
+  com_endpoint_config_key_chatgpt_copy_token: 'نسخ رمز الوصول.',
+  com_endpoint_config_key_google_need_to: 'أنت بحاجة إلى',
+  com_endpoint_config_key_google_vertex_ai: 'تمكين Vertex AI',
+  com_endpoint_config_key_google_vertex_api: 'API على Google Cloud، ثم',
+  com_endpoint_config_key_google_service_account: 'أنشئ حساب خدمة',
+  com_endpoint_config_key_google_vertex_api_role:
+   'تأكد من النقر على إنشاء ومتابعة" لمنح الدور "Vertex AI User" على الأقل. أخيرًا، قم بإنشاء مفتاح JSON للعمل على استيراده هنا.',
+  com_nav_auto_scroll: 'التمرير التلقائي إلى أحدث عند الفتح',
+  com_nav_plugin_store: 'متجر الإضافات',
+  com_nav_plugin_search: 'ابحث عن الإضافات',
+  com_nav_plugin_auth_error:
+   'حدث خطأ أثناء محاولة المصادقة على هذا البرنامج المساعد. يرجى المحاولة مرة أخرى.',
+  com_nav_close_menu: 'إغلاق القائمة الجانبية',
+  com_nav_open_menu: 'افتح القائمة الجانبية',
+  com_nav_export_filename: 'اسم الملف',
+  com_nav_export_filename_placeholder: 'قم بتعيين اسم الملف',
+  com_nav_export_type: 'النوع',
+  com_nav_export_include_endpoint_options: 'تضمين خيارات النقاط النهائية',
+  com_nav_enabled: 'تم التمكين',
+  com_nav_not_supported: 'غير مدعوم',
+  com_nav_export_all_message_branches: 'تصدير كل فروع الرسائل',
+  com_nav_export_recursive_or_sequential: 'التراجع أو التسلسل؟',
+  com_nav_export_recursive: 'تكراري',
+  com_nav_export_conversation: 'تصدير المحادثة',
+  com_nav_theme: 'المظهر',
+  com_nav_theme_system: 'النظام',
+  com_nav_theme_dark: 'داكن',
+  com_nav_theme_light: 'فاتح',
+  com_nav_clear_all_chats: 'مسح كل الدردشات',
+  com_nav_confirm_clear: 'تأكيد المسح',
+  com_nav_close_sidebar: 'إغلاق القائمة الجانبية',
+  com_nav_open_sidebar: 'افتح القائمة الجانبية',
+  com_nav_log_out: 'تسجيل الخروج',
+  com_nav_user: 'المستخدم',
+  com_nav_clear_conversation: 'مسح المحادثات',
+  com_nav_clear_conversation_confirm_message:
+   'هل أنت متأكد أنك تريد مسح جميع المحادثات؟ هذا لا يمكن التراجع عنه.',
+  com_nav_help_faq: 'مساعدة & الأسئلة الشائعة',
+  com_nav_settings: 'الإعدادات',
+  com_nav_search_placeholder: 'بحث في الرسائل',
+  com_nav_setting_general: 'عام',
+  com_nav_setting_data: 'تحكم في البيانات',
+  com_nav_language: 'اللغة',
+  com_nav_lang_auto: 'Auto detect',
+  com_nav_lang_english: 'English',
+  com_nav_lang_chinese: '中文',
+  com_nav_lang_german: 'Deutsch',
+  com_nav_lang_spanish: 'Español',
+  com_nav_lang_french: 'Français ',
+  com_nav_lang_italian: 'Italiano',
+  com_nav_lang_polish: 'Polski',
+  com_nav_lang_brazilian_portuguese: 'Português Brasileiro',
+  com_nav_lang_russian: 'Русский',
+  com_nav_lang_japanese: '日本語',
+  com_nav_lang_swedish: 'Svenska',
+  com_nav_lang_korean: '한국어',
+  com_nav_lang_traditionalchinese: '繁體中文',
+  com_nav_lang_arabic: 'العربية',
```

Comment:
can you please remove all theser translations?

Reply:
done